lib/resources/apache_conf.rb

```diff
@@ -40,6 +40,18 @@ def params(*opts)
     res
   end
 
+  def method_missing(name)
+    # ensure params are loaded
+    @params || read_content
+    
+    # extract values
+    param = @params[name.to_s]
+    return nil if param.nil?
+    # extract first value if we have only one value in array
+    return param[0] if param.length == 1
```

Comment:
as proposed by @srenatus we should not be smart here, lets leave this for the matcher